docs/source/commandline/commandline-all.rst

```diff
@@ -172,10 +213,57 @@ Example:
     front-ends and ROM managers), or processed with scripts query detailed
     information.
 
+    Example:
+        .. code-block:: bash
+
+            mame64 galaxian -listxml
+            <?xml version="1.0"?>
+            <!DOCTYPE mame [
+            <!ELEMENT mame (machine+)>
+	            <!ATTLIST mame build CDATA #IMPLIED>
+	            <!ATTLIST mame debug (yes|no) "no">
+	            <!ATTLIST mame mameconfig CDATA #REQUIRED>
+	            <!ELEMENT machine (description, year?, manufacturer?, biosset*, rom*, disk*, device_ref*, sample*, chip*, display*, sound?, input?, dipswitch*, configuration*, port*, adjuster*, driver?, feature*, device*, slot*, softwarelist*, ramoption*)>
+		            <!ATTLIST machine name CDATA #REQUIRED>
+		            <!ATTLIST machine sourcefile CDATA #IMPLIED>
+            ...
+            <mame build="0.216 (mame0216-154-gabddfb0404c-dirty)" debug="no" mameconfig="10">
+            	<machine name="galaxian" sourcefile="galaxian.cpp">
+		            <description>Galaxian (Namco set 1)</description>
+		            <year>1979</year>
+		            <manufacturer>Namco</manufacturer>
+                    ...
+            	<machine name="z80" sourcefile="src/devices/cpu/z80/z80.cpp" isdevice="yes" runnable="no">
+		            <description>Zilog Z80</description>
+	            </machine>
+            </mame>
+
+.. Tip:: Output from this command is typically more useful if redirected to
+         an output file. For instance, doing 
+         **mame64 -listxml galaxian > galax.txt** will make ``galax.txt`` or
```

Comment:
Please use an example filename with `.xml` extension to reinforce that it’s standard XML.

Reply:
Adjusted.